config/locales/views/exports/en.yml

```
@@ -1,14 +1,24 @@
 en:
-  export:
-    title: "Export"
+  exports:
+    index:
+      title: "Exports"
+      status: "Status"
+      filename: "Filename"
+      started: "Started"
+      ready_for_download: "Your export is ready for download"
+      created: "%{when} ago"
+      export_started: "We are preparing your export and will notify you when it's ready"
+      automatic_deletion: Created exports will be available for 30 days
+      explanation_html: "Here you can find all exports you created. Exports are an archive of the submissions you made in a course or series, or even <a href=\"%{user_link}\">all your submissions</a>. Teachers can also download the submissions that their students made in the courses they administrate. You can create an export of a series or a course by using its menu."
+      no_exports_yet_html: "It seems like you haven't created any exports yet, or they were deleted automatically after 30 days. Exports are an archive of the submissions you made in a course or series, or even <a href=\"%{user_link}\">all your submissions</a>. Teachers can also download the submissions that their students made in the courses they administrate. You can create an export of a series or a course by using its menu."
```

I would still display the explanation_html message and a shorter message instead of the table.
```suggestion
      no_exports_yet_html: "It seems like you haven't created any recent data archives. Get started and<a href=\"%{user_link}\">export your submissions</a>."
```